pkgs/development/python-modules/unicode-slugify/default.nix

```diff
@@ -18,15 +18,22 @@ buildPythonPackage rec {
     sha256 = "25f424258317e4cb41093e2953374b3af1f23097297664731cdb3ae46f6bd6c3";
   };
 
+  patches = [
+    ./use_pytest_instead_of_nose.patch
+    (fetchpatch {
+      url = "https://github.com/mozilla/unicode-slugify/commit/a18826f440d0b74e536f5e32ebdcf30e720f20d8.patch";
+      hash = "sha256-B27psp0XI5GhoR0l5lFpUOh88hHzjJYzJS5PnIkfFws=";
+    })
```

Comment:
Please link to the PR in a comment above so that people can still track the provenance of these changes.